pgml-dashboard/src/models.rs

```diff
@@ -549,8 +549,7 @@ impl Snapshot {
                     analysis,
                     created_at,
                     updated_at
-                FROM pgml.snapshots
-            "
+                FROM pgml.snapshots JOIN pg_class ON oid::regclass::text = relation_name"
```

Comment:
Does this work with views as well as tables?

Reply:
Yes, it was tested on both views and tables.